packages/integrations/netlify/test/edge-functions/edge-basic.test.ts

```diff
@@ -3,30 +3,38 @@ import { assertEquals, assert, DOMParser } from './deps.ts';
 
 Deno.env.set('SECRET_STUFF', 'secret');
 
-// @ts-expect-error
 Deno.test({
-	// TODO: debug why build cannot be found in "await import"
 	ignore: true,
 	name: 'Edge Basics',
-	skip: true,
-	async fn() {
+	permissions: 'inherit',
+	async fn(t) {
 		const fixture = loadFixture('./fixtures/edge-basic/');
-		await fixture.runBuild();
-		const { default: handler } = await import(
-			'./fixtures/edge-basic/.netlify/edge-functions/entry.js'
-		);
-		const response = await handler(new Request('http://example.com/'));
-		assertEquals(response.status, 200);
-		const html = await response.text();
-		assert(html, 'got some html');
+		await t.step('Run the build', async () => {
+			await fixture.runBuild();
+		});
+		await t.step('Should correctly render the response', async () => {
+			// TODO: debug why build cannot be found in "await import"
+			// Seems that the files are created, but can't be read at this stage.
+			// Remove the cleanup at the end and run twice, and the test will pass.
+			// This is weird...
+			const { default: handler } = await import(
+				'./fixtures/edge-basic/.netlify/edge-functions/entry.js'
+			);
```

Comment:
I noticed the test is now unskipped and CI is passing. Is the TODO here still needed?

I remember digging into the Deno tests here and found the same weird issues.

Reply:
I reverted the test command changes. I have no idea why Deno is not able to find the module, even it's there before the import execution, and the paths are correct.